runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsConfig.java

```diff
@@ -94,18 +94,31 @@ default boolean highlightTile()
 	@Alpha
 	@ConfigItem(
 		position = 4,
-		keyName = "markerColor",
-		name = "Marker color",
-		description = "Configures the color of object marker",
+		keyName = "boarderColor",
+		name = "Boarder color",
+		description = "Configures the boarder color of an object marker",
 		section = renderStyleSection
 	)
-	default Color markerColor()
+	default Color boarderColor()
 	{
 		return Color.YELLOW;
 	}
 
+	@Alpha
 	@ConfigItem(
 		position = 5,
+		keyName = "fillColor",
+		name = "Fill color",
+		description = "Configures the fill color of an object marker",
+		section = renderStyleSection
+	)
+	default Color fillColor()
+	{
+		return new Color(255, 255, 0, 0);
```

Comment:
I'd recommend giving the default fill a substantial amount of alpha, as object markers don't look good if they're filled with an opaque color.